<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrawingDialog</class>
 <widget class="QDialog" name="DrawingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>229</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>314</width>
    <height>136</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>11</y>
     <width>16</width>
     <height>207</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>52</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtnSet">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>54</y>
     <width>243</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="spinBoxR">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSlider" name="sliderR">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBoxG">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QSlider" name="sliderG">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBoxB">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QSlider" name="sliderB">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="phBtnCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>50</y>
     <width>80</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>182</y>
     <width>86</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Thickness:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSize">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>182</y>
     <width>55</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBoxR</tabstop>
  <tabstop>spinBoxG</tabstop>
  <tabstop>spinBoxB</tabstop>
  <tabstop>spinBoxSize</tabstop>
  <tabstop>pushBtnSet</tabstop>
  <tabstop>phBtnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>phBtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>DrawingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>installThinPen(QColor)</signal>
  <signal>installThickPen(QColor,int)</signal>
 </slots>
</ui>
